test/gurka/test_traffic.cc

```diff
@@ -87,7 +87,7 @@ TEST(Traffic, BasicUpdates) {
     auto result = gurka::route(map, "A", "C", "auto", {{"/date_time/type", "0"}}, clean_reader);
     gurka::assert::osrm::expect_steps(result, {"AB"});
     gurka::assert::raw::expect_path(result, {"AB", "BC"});
-    gurka::assert::raw::expect_eta(result, 14.303419);
+    gurka::assert::raw::expect_eta(result, 25.731991);
```

Comment:
So now an edge's speed can never be `kMaxTrafficSpeed` again. Is that gonna be a problem for traffic stuff?

Reply:
that is a very interesting question! the practical answer is no we likely dont care if a speed is above 87mph though we did used to allow it. i guess it would be nicer to do hard enforcement of the top speed only if it were set by the user and not defaulted but i can't practically think of why we wouldnt want to make a reasonable assumption like you're not going to drive faster than 90mph. i know there some rare places like in UAE where you can do 160kph but this seems like a pretty extreme case. maybe it'd be enough to up the maxassumed to 160kph and call it a day... let me think about that

Reply:
As long as absolute maxspeed is not country-specifically implemented (not sure about this, VS Code is balking this morning, can't track code..), I guess that's reasonable to default to the highest maxspeed of any country. With the understanding that you can go up to 252 KPH by setting higher `top_speed`s, e.g. in Germany where there's often (unfortunately) no real maxspeed on motorways, so some freaks might choose to actually go 250 KPH in real life. Then let's make it at least harder for them to plan routes like that ;)

Reply:
on line 82, can you use `kMaxAssumedSpeed` in place of `MAX_TRAFFIC_SPEED_KPH`

Reply:
Thoughts on raising `kMaxAssumedSpeed` or we leave it be for now?

Reply:
well i tested it with 160 and its a 3-4% performance penalty. so i think we probably shouldnt do that. which leaves the question of whether its ok to clamp traffic speeds down to 140kph in the case that they are higher than that. my gut reaction is yes its ok. @PureW @danpat any opinions on this? if traffic data says the road speed is >140kph should we actually prescribe that people drive that fast. my opinion is no with the caveat that people can opt into it by setting a top_speed = 160 or higher.